crates/wasmtime/src/runtime/vm.rs

```diff
@@ -200,6 +200,21 @@ pub unsafe trait VMStore {
     /// as a trap to clean up Wasm execution.
     fn maybe_async_gc(&mut self, root: Option<VMGcRef>) -> Result<Option<VMGcRef>>;
 
+    /// TODO FITZGEN
+    fn maybe_async_grow_gc_heap(&mut self, bytes_needed: u64) -> Result<()>;
+
+    /// TODO FITZGEN
+    ///
+    /// TODO FITZGEN: not guaranteed to have space after this
```

Comment:
TODO

Reply:
And these should return an error if `bytes_needed` space wasn't actually free'd?